web-backend/metrics/src/model/mod.rs

```diff
@@ -26,7 +26,7 @@ impl Db {
         password: Option<&str>,
     ) -> Result<Self> {
         // get MongoDB parameters
-        let host = host.unwrap_or("mongo");
+        let host = host.unwrap_or("localhost");
```

Comment:
does `make` or `make fast` still work with this? We do pass the correct hostname in docker-compose: https://github.com/diem/whackadep/blob/main/docker-compose.yml#L30 but I'm not sure if this is being picked up anywhere :D

Reply:
rechecked that `make` works.

`analysis::test_analysis` throws server connection timeout with `mongo` as hostname.

Reply:
I think a new db connection is initiated [here](https://github.com/diem/whackadep/blob/d760937abb15de11ee94ece622e54cfa63d45b6f/web-backend/metrics/src/analysis.rs#L48-L52). I have double checked, `mongo` as hostname doesn't work.

If it works on your machine, one possible reason could be `mongodb` being installed locally which might help interpreting `mongo` command.

Reply:
So wait, does this change need to be reverted then, back to "mongo"?

Reply:
No at least on my machine "mongo" as host name does not work, throws server connection timeout error. 
If I change it to "localhost", it works, passes all 9 tests, and make also works fine.

Reply:
My understanding is that "localhost" should work on any other machine as well.